src/importexport/guitarpro/internal/gtp/gpconverter.cpp

```diff
@@ -864,19 +864,19 @@ void GPConverter::setUpGPScore(const GPScore* gpscore)
     }
     if (!gpscore->subTitle().isEmpty() || !gpscore->artist().isEmpty() || !gpscore->album().isEmpty()) {
         Text* s = Factory::createText(_score->dummy(), TextStyleType::SUBTITLE);
-        QString str;
+        mu::String str;
         if (!gpscore->subTitle().isEmpty()) {
             str.append(gpscore->subTitle());
         }
         if (!gpscore->artist().isEmpty()) {
             if (!str.isEmpty()) {
-                str.append("\n");
+                str.append(mu::String("\n"));
```

Comment:
needs just `u` and maybe needs add method `append(char16_t ch)` to `String`
(actualy, here conversion `const char* -> QString -> String`)